assets/img/webp-convert-directory.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+PARAMS=('-q 75 -m 6 -mt -af -progress')
+
+if [ $# -ne 0 ]; then
+  PARAMS=$@;
+fi
+
+cd $(pwd)
+
+shopt -s nullglob nocaseglob extglob
+
+for FILE in *.@(jpg|jpeg|tif|tiff|png); do 
+    cwebp $PARAMS "$FILE" -o "${FILE%.*}".webp;
+done
```

Comment:
Would be nice to have some explanation to what is happening here before people run random bash scripts on their machines